cypress/support/pages/signUp.pageObject.js

```diff
@@ -0,0 +1,53 @@
+import PageObject from '../PageObject';
+
+class SignUpPageObject extends PageObject {
+    url = '/#/register';
+
+    get usernameField() {
+        return cy.getByDataCy('username-sign-up');
+      }
+
+    get emailField() {
+        return cy.getByDataCy('email-sign-up');
+      }
+    
+      get passwordField() {
+        return cy.getByDataCy('password-sign-up');
+      }
+    
+      get signUpBtn() {
+        return cy.getByDataCy('sign-up-btn');
+      }
+
+      get signUpUrl() {
+        return cy.url();
+      }
+    
+      typeUsername(email) {
+        this.usernameField
+          .type(email);
```

Comment:
The `typeUsername` method incorrectly uses the parameter `email`. It should be `username` to correctly reflect the purpose of typing a username.